incubator/group/keeper_test.go

```diff
@@ -19,14 +22,15 @@ import (
 func TestCreateGroup(t *testing.T) {
 	amino := codec.New()
 	pKey, pTKey := sdk.NewKVStoreKey(params.StoreKey), sdk.NewTransientStoreKey(params.TStoreKey)
-	paramSpace := subspace.NewSubspace(amino, pKey, pTKey, DefaultParamspace)
+	paramSpace := subspace.NewSubspace(amino, pKey, pTKey, group.DefaultParamspace)
 
-	groupKey := sdk.NewKVStoreKey(StoreKeyName)
-	k := NewGroupKeeper(groupKey, paramSpace, baseapp.NewRouter(), &MockProposalI{})
-	ctx := NewContext(pKey, pTKey, groupKey)
-	k.setParams(ctx, DefaultParams())
+	groupKey := sdk.NewKVStoreKey(group.StoreKeyName)
+	k := group.NewGroupKeeper(groupKey, paramSpace, baseapp.NewRouter(), &group.MockProposalI{})
+	ctx := group.NewContext(pKey, pTKey, groupKey)
+	defaultParams := group.DefaultParams()
+	paramSpace.SetParamSet(ctx, &defaultParams)
```

Comment:
```suggestion
	k.setParams(ctx, group.DefaultParams())
```

Reply:
This is not possible as the test is in the `group_test` package and has not access to the package private method.

Reply:
IMO we should export `setParams` and avoid paramSpace calls directly.